locations/spiders/hobby_lobby.py

```diff
@@ -0,0 +1,44 @@
+from urllib.parse import urlparse
+
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class HobbyLobbySpider(SitemapSpider, StructuredDataSpider):
+    name = "hobby_lobby"
```

Comment:
This brand only seems to exist in the United States so the spider `name` should be `hobby_lobby_us` and the class name should be `HobbyLobbyUSSpider`. The file should be renamed to match the spider `name`.